packages/app/src/components/collapsible/collapsible-section.tsx

```diff
@@ -115,8 +115,18 @@ export const CollapsibleSection = ({
    * If so, the collapsible needs to be opened.
    */
   useEffect(() => {
-    const isOpenedByQueryParam = window.location.hash.substr(1) === id;
-    setOpen(isOpenedByQueryParam);
+    const onHashChange = function () {
+      const isOpenedByQueryParam = window.location.hash.substr(1) === id;
```

Comment:
nit; I think this is a little more readable. Param is kind of implied by the use of query already.

```suggestion
      const isOpenedByQuery = window.location.hash.includes(id);
```

Reply:
I think the current method is most resilient. Using `includes()` or even `endsWith()` will be risky when IDs have overlapping parts. In the case of `includes()`, if you go to `http://localhost:3000/verantwoording#sterftemonitor-cbs` the 'Sterfte' section will also open, for example.

Reply:
👍

Reply:
Agree. For readability this is also an option:
```suggestion
      const isOpenedByQueryParam = window.location.hash === `#${id}`;
```

Reply:
Even that would not be functionally equivalent. This would open all collapsibles without an `id` if the hash would be literally '#undefined', while the former also protects against that particular case.

Then again, I don't think a hash will ever be 'undefined', so I think we can go for readability here.